example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+Plotting CO-Values using deck.gl
+================================
+
+In this example we explain how to use [deck.gl](https://deck.gl/#/),
```

Comment:
@tibroc Why are hyphens better than commas? And where would I find it on my German keyboard? ;)

Reply:
For consistency, you actually want an en-dash and no hyphen. The location depends on your keyboard layout. For the `nodeadkeys` variant of the German keyboard layout, you can use `Alt Gr` + `-`. With a compose key, this would be `<Compose - - .` (Fedora default settings).

Reply:
So, if we agree on en-dashes (which would be okay for me), I find that they distinguish the insertion more clearly from the main phrase which – at least for me – improves legibility. I would see the main effect using actual parentheses, although in a more timid way.

Reply:
I copied @lkiesow's en-dash and changed it.